### ISSUE TITLE
Lastmod Property for Page Resources

### ISSUE THREAD

**Issue by sebastianw** [gohugoio/hugo#9781]
Currently only Pages can have a `.Lastmod` property from `.GitInfo`. It would be great to have the same information for page resources. This would help for pages that mostly display a prominent resource (for example an image). Then e.g. the last modified date of the page in the sitemap could be generated from that resource with a custom template.

**Comment by jmooring** [comment]
I know this is not what you are asking for, but it might be helpful.

```
{{ with .Resources.Get "a.jpg" }}
  {{ (os.Stat .Key).ModTime }}
{{ end }}
```

**Comment by sebastianw** [comment]
> I know this is not what you are asking for, but it might be helpful.

Yes, thanks! Problem is that I use hugo in a pipeline so the modification time is the time it is checked out from git which is the current time. This might be helpful for other situations though!

**Comment by bep** [comment]
@sebastianw I agree that we should do this; there is a related issue about some of the limitations with the current GitInfo implementation (it doesn't work in a multi module setup, for one), and this issue would be a natural extension to that.
